### ISSUE TITLE
Wishlist: Support for Feedly tagging

### ISSUE THREAD

**Issue by mcackay** [phantomfish/byline#14]
Would be cool to be able to tag articles of interest -- e.g.  a video I want to watch later, an article which maybe good for my organisation's newsletter, etc.

**Comment by phantomfish** [comment]
Yes, tags aren't that widely used but it'd be nice to support them. I doubt this will make 5.0 though.
